### PR TITLE
Update samples test to support project_national

### PR DESCRIPTION
**Pull request by joseph-robertson**
Resolves #[issue number here].

## Pull Request Description

[description here]

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [ ] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [ ] All tests are passing
- [ ] The [changelog](https://github.com/NREL/resstock/blob/develop/CHANGELOG.md) has been updated appropriately
- [ ] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

File: resources/buildstock.rb [hunk 1]
```diff
@@ -450,6 +462,12 @@ def self.get_build_existing_model(result, rows)
   end
 
   def self.get_simulation_output_report(result, rows)
+    rows['SimulationOutputReport'].each do |k, v|
+      begin
+        rows['SimulationOutputReport'][k] = v.round(1)
```
Comment (joseph-robertson) [inline resources/buildstock.rb:467]: Make sure this rounding is appropriate for all outputs.

File: test/test_measures_osw.rb [hunk 2]
```diff
@@ -7,16 +7,17 @@
 
 class TestResStockMeasuresOSW < MiniTest::Test
   def test_measures_osw
-    project_dir = 'project_testing'
-    num_samples = 10
+    project_dir = 'project_national'
```
Comment (joseph-robertson) [inline test/test_measures_osw.rb:10]: Should do testing on `project_testing` as well.

**Comment by shorowit** [comment]
Shouldn't there be two results csv files? (Or shouldn't the results csv have the results from both testing and national projects?)

**Comment by joseph-robertson** [comment]
There could be. I opted to just keep the one corresponding to the national project.

**Pull request merged**